### PR TITLE
Update README.md to include more setup information

### PR DESCRIPTION
**Pull request by twiest**
- [x] Change the README.md to include instructions to setup the more reliable json-file logger
- [x] Change the README.md to include Fedora 27 package requirements
- [x] Change the README.md to include instructions to setup the system to work with kubernetes ansible modules
- [x] Test the new instructions to ensure they work as expected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@
       * Alternatively, you can also compile `oc` from source.
       * Note: It is recommended to put the `oc` binary somewhere in your path.
     * Mac OSX: [Minishift](https://github.com/minishift/minishift/releases) is the recommended development environment
+  * Create a `kubectl` symlink to the `oc` binary (Note: It is recommended to put the `kubectl` symlink somewhere in your path.):
+    * `ln -s oc kubectl`
```
Comment (dgoodwin) [inline README.md]: What is the rationale behind this one?

Comment (twiest) [reply README.md]: If you don't have it, then the playbook fails (at least it did for @joelddiaz and me).

@joelddiaz said it's specifically the `kubectl_apply` module requires it.

**Comment by twiest** [comment]
@dgoodwin Thanks for the feedback. I believe I've addressed your comments. Please re-review.

**Pull request merged**